### PR TITLE
optim bazel

### PR DESCRIPTION
**Pull request by wu-hanqing**
- add `.bazelversion` to support bazelisk
- add `.bazelignore` and exclude `cmake_third_party` to fix [this](https://github.com/trpc-group/trpc-cpp/blob/main/docs/zh/faq/bazel_and_cmake_problem.md#%E4%BD%BF%E7%94%A8cmake%E7%BC%96%E8%AF%91%E5%90%8E%E5%86%8D%E4%BD%BF%E7%94%A8bazel%E7%BC%96%E8%AF%91%E5%AF%BC%E8%87%B4%E6%8A%A5%E9%94%99)

### CODE REVIEW

**Comment by lioncfliu** [comment]
Thank you for contributing code, please add  a comment with "I have read the CLA Document and I hereby sign the CLA" to sign CLA. 

e.g.
image

File: BUILD.bazel
Comment (weimch) [inline BUILD.bazel:1]: All BUILD files in this project are bazel BUILD files. The meaning is clear. And we won't define another BUILD file that is not a bazle BUILD file.

Is there exists any special advantage to change filename from BUILD into BUILD.bazel ?

Comment (wu-hanqing) [reply BUILD.bazel:1]: Sorry for the late reply. Of course, bazel supports both forms, but on some MacOS platforms, the file system is not case-sensitive, which can result in not being able to create a `build` folder in the working directory.

If this is not a problem, I will drop this modification.

Comment (weimch) [reply BUILD.bazel:1]: Sorry for not checking the email notification in time when your replied.

tRPC-Cpp has no plan to support other platforms expect linux. This modification seems no need.

**Comment by wu-hanqing** [comment]
I have read the CLA Document and I hereby sign the CLA

File: BUILD.bazel
Comment (wu-hanqing) [reply BUILD.bazel:1]: > Sorry for not checking the email notification in time when your replied.
> 
> tRPC-Cpp has no plan to support other platforms expect linux. This modification seems no need.

Fixed

**Pull request closed without merging**